content/enterprise/before-installing/reference-architecture/azure.mdx

```diff
@@ -41,7 +41,7 @@ Depending on the chosen [operational
 mode](/enterprise/before-installing/#operational-mode-decision),
 the infrastructure requirements for Terraform Enterprise range from a single [Azure VM
 instance](https://azure.microsoft.com/en-us/services/virtual-machines/) for
-demo or proof of concept installations to multiple instances connected to
+_Mounted Disk_ installations to multiple instances connected to
```

Comment:
"Requirements depend on the chosen operational mode. For example, an installation on Mounted Disk mode may require a single Azure VM instance, whereas a stateless production installation may require multiple instances connected to Azure Database for PostgreSQL and Azure Blob Storage."